pkg/services/apiserver/aggregator/README.md

```diff
@@ -45,3 +45,52 @@ live under that instead.
 `service.grafana.app`) and `data/grafana-apiextensions` (`apiextensions.k8s.io`).
 2. Since `grafana-aggregator` outputs configuration (TLS and kubeconfig) that is used in the invocation of aggregated
   servers, ensure you start the aggregated service after launching the aggregator during local development.
+
+## How to get started
```

Comment:
@ryantxu i added docs and an example based on what @charandas wrote for the standalone aggregator command. remote seems to work as expected:

```shell
export KUBECONFIG=$PWD/data/grafana-apiserver/grafana.kubeconfig
kubectl get apiservices
NAME                                 SERVICE                     AVAILABLE   AGE
v0alpha1.example.grafana.app         grafana/example-apiserver   True        3m42s
v0alpha1.featuretoggle.grafana.app   Local                       True        73m
v0alpha1.playlist.grafana.app        Local                       True        73m
v0alpha1.service.grafana.app         Local                       True        73m
```

```shell
todd@work ~/g/s/g/g/grafana (toddtreece/standalone-aggregation) [127]> kubectl api-resources
NAME             SHORTNAMES   APIVERSION                           NAMESPACED   KIND
apiservices                   apiregistration.k8s.io/v1            false        APIService
dummy                         example.grafana.app/v0alpha1         true         DummyResource
runtime                       example.grafana.app/v0alpha1         false        RuntimeInfo
features                      featuretoggle.grafana.app/v0alpha1   false        Feature
featuretoggles                featuretoggle.grafana.app/v0alpha1   true         FeatureToggles
playlists                     playlist.grafana.app/v0alpha1        true         Playlist
externalnames                 service.grafana.app/v0alpha1         true         ExternalName
```